pkg/pcp/manifest.json.in

```diff
@@ -5,7 +5,7 @@
     "bridges": [
         {
             "match": { "payload": "metrics1" },
-            "spawn": [ "@libexecdir@/cockpit-pcp" ]
+            "spawn": [ "${libexecdir}/cockpit-pcp" ]
```

Comment:
This needs to bump the `requires.cockpit` to 238.x. This is using an API that is just being introduced into cockpit-bridge and not released yet, but the .x trick should work with our builds from master.

This *should* do the right thing to cockpit-pcp's rpm and deb package dependencies, which use the `required_base` and auto-computation. Let's make double-sure that this works as intended.

Reply:
thanks!